src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -329,4 +329,43 @@ class JavaMigrations {
       .validate()
       .insert()
   }
+
+  @ChangeSet(order = "035", id = "035-add_zulu_6_0_113", author = "vpavic")
+  def migrate035(implicit db: MongoDatabase) = {
+    Seq(Linux64, Windows).foreach(platform => removeVersion(candidate = "java", version = "6.0.107-zulu", platform))
+    List(
+      Version("java", "6.0.113-zulu", "https://cdn.azul.com/zulu/bin/zulu6.21.0.3-jdk6.0.113-linux_x64.tar.gz", Linux64),
+      Version("java", "6.0.113-zulu", "https://cdn.azul.com/zulu/bin/zulu6.21.0.3-jdk6.0.113-win_x64.zip", Windows)
+    ).validate().insert()
+  }
+
+  @ChangeSet(order = "036", id = "036-add_zulu_7_0_191", author = "vpavic")
+  def migrate036(implicit db: MongoDatabase) = {
+    Seq(Linux64, Windows, MacOSX).foreach(platform => removeVersion(candidate = "java", version = "7.0.181-zulu", platform))
```

Comment:
I think we should probably leave 7.0.181-zulu for OSX in tact as it still seems to be working as a tarball. We can replace it with 7.0.191 for the other platforms.

Reply:
Whoops - I missed that one, you're right, of course.